diff --git a/config/user_aliases.cmd b/config/user_aliases.cmd
index 45f44a7..d68d937 100644
--- a/config/user_aliases.cmd
+++ b/config/user_aliases.cmd
@@ -12,3 +12,8 @@ clear=cls
 unalias=alias /d $1
 vi=vim $*
 cmderr=cd /d "%CMDER_ROOT%"
+cd~=cd %HOMEPATH%
+rmtmp=rm -rf %TEMP%/*
+pecog=peco-tool.bat ghq
+pecoh=peco-tool.bat history
+pecohc=peco-tool.bat history clip
diff --git a/vendor/clink.lua b/vendor/clink.lua
index 4ba4dd6..8ae83d0 100644
--- a/vendor/clink.lua
+++ b/vendor/clink.lua
@@ -5,7 +5,7 @@
 
 -- luacheck: globals clink
 
--- At first, load the original clink.lua file
+-- At first, load the original clink.lua fil
 -- this is needed as we set the script path to this dir and therefore the original
 -- clink.lua is not loaded.
 local clink_lua_file = clink.get_env('CMDER_ROOT')..'\\vendor\\clink\\clink.lua'
@@ -47,8 +47,8 @@ local function set_prompt_filter()
     -- build our own prompt
     -- orig: $E[1;32;40m$P$S{git}{hg}$S$_$E[1;30;40m{lamb}$S$E[0m
     -- color codes: "\x1b[1;37;40m"
-    local cmder_prompt = "\x1b[1;32;40m{cwd} {git}{hg}{svn} \n\x1b[1;39;40m{lamb} \x1b[0m"
-    local lambda = "λ"
+    local cmder_prompt = "\x1b[1;32;49m{cwd} {git}{hg}{svn} \n\x1b[1;39;49m{lamb} \x1b[0m"
+    local lambda = "$"
     cmder_prompt = string.gsub(cmder_prompt, "{cwd}", verbatim(cwd))
 
     if env ~= nil then
@@ -302,8 +302,8 @@ local function git_prompt_filter()
 
     -- Colors for git status
     local colors = {
-        clean = "\x1b[1;37;40m",
-        dirty = "\x1b[33;3m",
+        clean = "\x1b[1;37;49m",
+        dirty = "\x1b[33;49m",
         conflict = "\x1b[31;1m"
     }
 
@@ -347,7 +347,7 @@ local function hg_prompt_filter()
     if hg_dir then
         -- Colors for mercurial status
         local colors = {
-            clean = "\x1b[1;37;40m",
+            clean = "\x1b[1;37;49m",
             dirty = "\x1b[31;1m",
         }
 
@@ -381,7 +381,7 @@ end
 local function svn_prompt_filter()
     -- Colors for svn status
     local colors = {
-        clean = "\x1b[1;37;40m",
+        clean = "\x1b[1;37;49m",
         dirty = "\x1b[31;1m",
     }
 
